### ISSUE TITLE
Add rule for two empty lines between classes

### ISSUE THREAD

**Issue by humdingerb** [owenca/haiku-format#1]
E.g between is just one, which isn't corrected at the moment:
```
App::App()
	:
	BApplication("application/x-vnd.Haiku-Haiku3d"),
	fMainWindow(NULL)
{
}

App::~App()
{
}
```

**Comment by owenca** [comment]
Better leave it alone when there are comments between the top-level constructs. Too many varieties to let the formatter figure out which comment blocks are meant for which construct.

**Comment by owenca** [comment]
Fixed in 10.0.1.
